platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -2703,16 +2840,19 @@ namespace platoon_strategic_ihp
         fmter %pose_ecef_point_.ecef_x;     //  index = 2
         fmter %pose_ecef_point_.ecef_y;     //  index = 3
         fmter %pose_ecef_point_.ecef_z;     //  index = 4
-             
-        fmter %target_join_index_;             //  index = 5
+        //Note: remove join_index to STATUS param. request.join_index = target_join_index_;
 
         request.strategy_params = fmter.str();
-                
+        
         mobility_request_publisher_(request); 
         ROS_DEBUG_STREAM("Published Mobility cut-in join request to the leader");
         ROS_WARN("Published Mobility cut-in join request to the leader");
     }
 
+    // UCLA: run prepare to depart state for depart from platoon
```

Comment:
I suspect that there is no transition out of the PREPARETOJOIN state other than for exception situations.  I would expect to see it in the run_prepare_to_join() method and normally transition to FOLLOWER or CANDIDATEFOLLOWER.  Maybe I'm looking in the wrong place?